### PR TITLE
Feature rebuild notifications interface

### LINKED ISSUE #490: Update the look of the notifications page
There is no mockup for this page

Where is fronted designer when he is needed?

### PR DESCRIPTION
**Pull request by Karrp**
Resolve #490

### CODE REVIEW

File: users/templates/user_notifications_list.html [hunk 1]
```diff
@@ -1,36 +1,46 @@
 {% extends 'base.html' %}
 
 {% load i18n %}
+{% load static %}
 {% load data_structure_element_selectors %}
 
+{% block extra_head %}
+  <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" type="text/css" href="{% static 'users/style.css' %}">
```
Comment (kbeker) [inline users/templates/user_notifications_list.html]: All these stylesheets does not have SRI.

File: users/templates/user_notifications_list.html [hunk 2]
```diff
@@ -1,36 +1,46 @@
 {% extends 'base.html' %}
```
Comment (kbeker) [inline users/templates/user_notifications_list.html]: Please stick to our convention with 4 indents.

File: users/static/users/style.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    width: 50px;
+    text-align: center;
+}
+
+.email-header {
+    width: 200px;
+    text-align: center;
+}
+
+
```
Comment (kbeker) [inline users/static/users/style.css]: Redundant blank line

File: users/templates/user_notifications_list.html (hunk 2 above)
Comment (Karrp) [reply users/templates/user_notifications_list.html]: done

File: users/templates/user_notifications_list.html (hunk 1 above)
Comment (Karrp) [reply users/templates/user_notifications_list.html]: done

File: users/static/users/style.css (hunk 3 above)
Comment (Karrp) [reply users/static/users/style.css]: done

File: users/templates/user_notifications_list.html [hunk 4]
```diff
@@ -1,36 +1,60 @@
...
+        integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+        crossorigin="anonymous"
+    />
+    <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+        integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+        crossorigin="anonymous"
+    />
+    <link
```
Comment (Szymiks) [inline users/templates/user_notifications_list.html]: This seems to be unnecessary because we are using font awesome in `base`

Comment (kbeker) [reply users/templates/user_notifications_list.html]: @Szymiks is right, please extract this stylesheet to template from which both place inherit and don't repeat loading this script again.

Comment (Karrp) [reply users/templates/user_notifications_list.html]: Done

File: users/templates/user_notifications_list.html [hunk 5]
```diff
@@ -1,36 +1,56 @@
...
+        rel="stylesheet"
+        href="https://code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css"
+        integrity="sha384-Nlo8b0yiGl7Dn+BgLn4mxhIIBU6We7aeeiulNCjHdUv/eKHx59s3anfSUjExbDxn"
+        crossorigin="anonymous"
+    />
+    <link
+        rel="stylesheet"
+        type="text/css"
+        href="{% static 'users/style.css' %}"
+    />
```
Comment (kbeker) [inline users/templates/user_notifications_list.html]: @Karrp please, if you do something first see what have be done before. Please add  subresource integrity also here.

Comment (Karrp) [reply users/templates/user_notifications_list.html]: Done

**Pull request merged**